crates/bevy_gilrs/src/converter.rs

```diff
@@ -40,6 +40,7 @@ pub fn convert_axis(axis: gilrs::Axis) -> Option<GamepadAxisType> {
         // The `axis_dpad_to_button` gilrs filter should filter out all DPadX and DPadY events. If
         // it doesn't then we probably need an entry added to the following repo and an update to
         // GilRs to use the updated database: https://github.com/gabomdq/SDL_GameControllerDB
-        gilrs::Axis::Unknown | gilrs::Axis::DPadX | gilrs::Axis::DPadY => None,
+        gilrs::Axis::Unknown => Some(GamepadAxisType::Other(code.into_u32() as u8)),
```

Comment:
code representation is platform specific and can rarely be usefully converted to a `u8`
* on macOS, it's two `u32` https://gitlab.com/gilrs-project/gilrs/-/blob/master/gilrs-core/src/platform/macos/gamepad.rs#L413
* on windows, it's enum and an `u32` https://gitlab.com/gilrs-project/gilrs/-/blob/master/gilrs-core/src/platform/windows_wgi/gamepad.rs
* on linux, it's two `u16` https://gitlab.com/gilrs-project/gilrs/-/blob/master/gilrs-core/src/platform/linux/gamepad.rs#L791
* on wasm, it is a `u8` https://gitlab.com/gilrs-project/gilrs/-/blob/master/gilrs-core/src/platform/wasm/gamepad.rs#L310

Anyone relying on those value will lose compatibility across platforms, and is at high risk of collisions between inputs

Reply:
I went ahead and pushed a change bumping that field up to u32, size between `GamepadButtonType::Other` and `GamepadAxisType::Other` now differs as ButtonType is left at u8.

I'm really not too fond of this arbitrarily packed `code` value, though I suppose it is better than no events?